docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/index.md

```diff
@@ -273,6 +307,23 @@ In addition the mobile package has some contexts that can be enabled depending o
+| `snowplow__ecommerce_event_names`       | `['snowplow_ecommerce_action']`         | The list of event names that the Snowplow e-commerce package will filter on when extracting events from your atomic events table. If you have included any custom e-commerce events, feel free to add their event name in this list to include them in your data models.                                                                                                                                                                                                   |
+| `snowplow__max_session_days`            | 3                                       | The maximum allowed session length in days. For a session exceeding this length, all events after this limit will stop being processed. Exists to reduce lengthy table scans that can occur due to long sessions which are usually a result of bots.                                                                                                                                                                                 |
+| `snowplow__number_category_levels`      | `4`                                     | The maximum number of levels (depth) of subcategories that exist on your website for products. These subcategories are recorded in the category field of the product context, and should be separated using the separator which is defined below. For example, `books/fiction/magical-fiction` has a level of 3. The value is the number of columns that will be generated in the product tables created by this Snowplow dbt package. |
+| `snowplow__number_checkout_steps`       | `4`                                     | The index of the checkout step which represents a completed transaction. This is required to enable working checkout funnel analysis, and has a default value of 4.                                                                                                                                                                                                                                       |
+| `snowplow__user_product_quantity`       | `false`                                 | Whether the `product_quantity` field in the product context should be used to sum up the total number of products in a transaction. If this value is set to false, then your `number_products` field in your `transaction` tables will instead be calculated by counting the number of product entities within the transaction i.e. treating each product as having a quantity of 1. |
+
```

Comment:
Can you add _all_ variables to the config, even those covered in other areas of the docs. Also remove the `snowplow__` prefix as it takes up too much space.

For some reason as well the table isn't rendering right in the preview but I can't see why...

Reply:
It should be fixed now!